site/sfguides/src/build-a-cinebot-with-snowflake-native-apps-and-cortex/build-a-cinebot-with-snowflake-native-apps-and-cortex.md

```diff
@@ -0,0 +1,255 @@
+author: Charles Yorek
+id: build-a-cinebot-with-snowflake-native-apps-and-cortex
+summary: This is a sample Snowflake Guide
+categories: Getting-Started
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Native Apps, Cortex, Data Applications
+
+# Build a CineBot with Snowflake Native Apps and Cortex
+<!-- ------------------------ -->
+## Overview
+Duration: 1
+
+The Snowflake Native App Framework is a powerful way for application providers to build, deploy and market applications via the Snowflake Marketplace.  In this example you will learn how to incorporate Snowflake's Cortex Suite into a Native App that will create a 'CineBot' that can recommend Movies. 
+
+### Prerequisites
```

Comment:
@sfc-gh-cyorek Add link to Snowflake signup/signin page.

Reply:
Added